docs/api-guide.md

```diff
@@ -216,177 +237,193 @@ Set<String> namespaces = admin.getNamespaceNames();
 You can get the tables of a namespace as follows:
 
 ```java
-// Get the tables of the namespace "ns"
+// Get the tables of the namespace "ns".
 Set<String> tables = admin.getNamespaceTableNames("ns");
 ```
 
-### Get a table metadata
+### Get table metadata
 
-You can get a table metadata as follows:
+You can get table metadata as follows:
 
 ```java
-// Get a table metadata of "ns.tbl"
+// Get the table metadata for "ns.tbl".
 TableMetadata tableMetadata = admin.getTableMetadata("ns", "tbl");
 ```
 
-### Operations for Coordinator tables
+### Specify operations for Coordinator tables
+
+Coordinator tables are used by the [Transactional API](#transactional-api) to track the statuses of transactions.
```

Comment:
Actually, we use plural forms in APIs, such as `admin.createCoordinatorTables()`, because I anticipated that we would have multiple coordinator tables in the future, like lock tables for `EXTRA_WRITE`, in addition to the state table. For the sake of consistency, should we continue to use plural forms?

Reply:
I only saw one Coordinator table when running ScalarDB, but I assumed there were multiple after seeing APIs like `admin.createCoordinatorTables()`, which is why I left `Coordinator tables` plural.

@brfrn169 If we currently only have one type of Coordinator table, I think we should use the singular form to avoid confusion. Is that OK?

Reply:
Oh, I didn't know the API uses plural.
Then, I think we should note it since the method name and the actual number of tables are inconsistent; otherwise, it is misleading.

One way is we call it `Coodinator table` and note that the method name is plural for future extension.
The other way is we call it `Coordinator tables` and note that we only have one table at the moment, but it can be extended.

@brfrn169 @josh-wong Thought?

Reply:
> One way is we call it `Coordinator table` and note that the method name is plural for future extension.

I'd prefer this way to reduce confusion. @brfrn169, is this way OK with you?

Reply:
@josh-wong Yes, that's fine with me too. Thanks.

Reply:
@brfrn169 (cc: @feeblefakie) Thank you for confirming! How does the following note sound?:

```suggestion
The Coordinator table is used by the [Transactional API](#transactional-api) to track the statuses of transactions.

{% capture notice--info %}
**Note**

Although only one Coordinator table can be created, the name of the API method is plural for future extension.
{% endcapture %}

<div class="notice--info">{{ notice--info | markdownify }}</div>
```

Reply:
@josh-wong Sounds good. Thank you!